app/internal/page/app.go

```diff
@@ -458,17 +461,21 @@ func makeHandle(mux *http.ServeMux, logger Logger, store sessions.Store, localiz
 
 			_, cookieErr := r.Cookie("cookies-consent")
 
+			localizer.ShowTranslationKeys = r.FormValue("showTranslationKeys") == "1" && !isProduction
+
 			if err := h(AppData{
-				Page:             path,
-				Query:            queryString(r),
-				Localizer:        localizer,
-				Lang:             lang,
-				SessionID:        sessionID,
-				CookieConsentSet: cookieErr != http.ErrNoCookie,
-				CanGoBack:        opt&CanGoBack != 0,
-				RumConfig:        rumConfig,
-				StaticHash:       staticHash,
-				Paths:            paths,
+				Page:                path,
+				Query:               queryString(r),
+				Localizer:           localizer,
+				Lang:                lang,
+				SessionID:           sessionID,
+				CookieConsentSet:    cookieErr != http.ErrNoCookie,
+				CanGoBack:           opt&CanGoBack != 0,
+				RumConfig:           rumConfig,
+				StaticHash:          staticHash,
+				Paths:               paths,
+				IsProduction:        isProduction,
+				ShowTranslationKeys: r.FormValue("showTranslationKeys") == "1",
```

Comment:
I didn't think when I commented before, you have `Localizer` in here so could just use `.App.Localizer.ShowTranslationKeys`